config/generic-ultimaker-ultimainboard-v2.cfg

```diff
@@ -74,6 +76,14 @@ max_temp: 275
 #min_temp: 0
 #max_temp: 275
 
+[heater_fan nozzle_fan]
```

Comment:
Please consider this naming, as [fan] is the parts fan, and the naming you have chosen for the heater_fan can be confusing, we have settled on `[heater_fan heatbreak_cooling_fan]` as the common naming for a fan that cools the hot end. Unless this is some other type of fan that isnt the main parts fan or the heatbreak cooling fan?

Thanks
James

Reply:
Updated.